sdk/go/src/sawtooth_sdk/client/encoder.go

```diff
@@ -243,6 +246,12 @@ func Decode(encoded, encoding string) []byte {
 			panic("Couldn't decode")
 		}
 		return bytes
+	case "base64":
+		bytes, err := base64.StdEncoding.DecodeString(encoded)
+		if err != nil {
+			panic("Couldn't decode")
```

Comment:
This Decode() func seems to do the same thing as base64.StdEncoding.DecodeString() only panic instead of returning an error.  Seems like we should instead just use the libraries directly so clients can handle the errors.